sampler.go

```diff
@@ -253,10 +365,10 @@ func NewRemotelyControlledSampler(
 }
 
 // IsSampled implements IsSampled() of Sampler.
-func (s *RemotelyControlledSampler) IsSampled(id uint64) bool {
+func (s *RemotelyControlledSampler) IsSampled(id uint64, operation string) bool {
```

Comment:
The remote sampler will generate an adaptive sampler if the operation sampling rates are passed in.